pdc/apps/package/models.py

```diff
@@ -27,6 +27,7 @@ class RPM(models.Model):
     release             = models.CharField(max_length=200, db_index=True)
     arch                = models.CharField(max_length=200, db_index=True)  # nosrc
     srpm_name           = models.CharField(max_length=200, db_index=True)  # package (name of srpm)
+    koji_build_target   = models.CharField(max_length=200, null=True, blank=True, db_index=True)
```

Comment:
I don't think this field belongs here.
PDC should be about *metadata* storage.
It shouldn't refer to any systems or services (with the exception of bindings).
I propose to move this field outside RPM model to some external module or extension and map it via OneToOne relation.